manifests/init.pp

```diff
@@ -216,6 +219,7 @@
   Array[Stdlib::Absolutepath] $allowed_import_path = ['/var/lib/pulp/sync_imports'],
   Array[Stdlib::Absolutepath] $allowed_export_path = [],
   Pulpcore::ChecksumTypes $allowed_content_checksums = ['sha224', 'sha256', 'sha384', 'sha512'],
+  Integer[1] $keep_changelog_limit = undef,
```

Comment:
This is not a valid type. It should be
```suggestion
  Optional[Integer[1]] $keep_changelog_limit = undef,
```

That said, the feature is explicit in that `0` should be the lowest value. See https://github.com/pulp/pulp_rpm/commit/cbc5e98770da6469bed936070b4f1c07fb5835c8#